src/components/PeopleTable.tsx

```diff
@@ -126,9 +127,12 @@ export const PeopleTable: React.FC<Props> = ({ people }) => {
           >
             <td>
               <Link
-                to={`../${person.slug}`}
+                to={{
+                  pathname: `../${person.slug}`,
+                  search: searchParams.toString(),
```

Comment:
now there are no changes on deployed page

Please, do not forget to re-deploy solution after every fix (before review) and make sure it works:
![image](https://github.com/mate-academy/react_people-table-advanced/assets/86923089/d889087e-4b2d-4d5a-a18f-11fbb4e2bc60)